src/main/java/uk/ac/ebi/eva/commons/models/mongo/documents/subdocuments/VariantAnnotation.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.mongo.documents.subdocuments;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+import org.springframework.util.Assert;
+import uk.ac.ebi.eva.commons.models.mongo.documents.Annotation;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Annotations of the genomic variation
```

Comment:
"Lite indexing version of Annotation generated using Ensembl VEP of a genomic variation." ?